packages/kit-bg/src/services/ServiceAccountProfile.ts

```
@@ -454,6 +455,19 @@ class ServiceAccountProfile extends ServiceBase {
         }
         result.walletAccountName = `${item.walletName} / ${item.accountName}`;
         result.walletAccountId = item.accountId;
+        if (enableAddressDeriveInfo) {
+          const account =
+            await this.backgroundApi.serviceAccount.getNetworkAccountsInSameIndexedAccountIdWithDeriveTypes(
+              {
+                networkId,
+                indexedAccountId: item.accountId,
+              },
+            );
+          if (account.networkAccounts && account.networkAccounts[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)